.dockerignore

```diff
@@ -3,3 +3,5 @@ node_modules
 packages/*/node_modules
 plugins/*/node_modules
 plugins/*/dist
+Dockerfile
+Dockerfile.standalone
```

Comment:
these are probably not intended to be here

Reply:
I added them to `.dockerignore` so changing the `Dockerfile.standalone`  would not invalidate docker build cache. (Because if a file in COPY is changed docker will not use build cache from that layer on)

But it is unnecessary for the original Dockerfile since it wont be copied to the docker image at all so I will remove that one.

Reply:
Think we could remove this one too tbh